### DISCUSSION TITLE
本文で脚注が連続すると分かりづらくなる

### DISCUSSION THREAD

**Discussion by yarakos95** [increments/qiita-discussions#816] [要望・アイデア・不具合]
現在、脚注番号はアラビア数字のみの “1” などとなっています。そのため、複数の脚注が連続していると数字も連続してしまうため、1 つの脚注のように見えてしまいます。

本文内で連続する脚注が複数あることが分かりやすくなっていると良いなと思います。

-----

以下の例では、本文内の脚注番号が 2 つあるにも関わらず、脚注番号 “12” が 1 つだけに見えてしまいます。

```markdown
[^1][^2]

[^1]: 1 つ目の脚注
[^2]: 2 つ目の脚注
```

![image](https://github.com/increments/qiita-discussions/assets/75878492/e5a29ef4-ebdd-49db-92e8-d8a86f3262b4 "脚注番号が 12 に見える例")

例えば GitHub ではアラビア数字が角括弧で囲われています。そのため、本文中で連続していても複数の脚注があることが分かります。

脚注の例：[^1][^2]

[^1]: 1 つ目の脚注
[^2]: 2 つ目の脚注

**Comment by tanaka-asami415** [comment]
@Yarakashi-Kikohshi 
ご意見とご提案をいただき、ありがとうござます！
確かに分かりにくいので修正いたします🙇

**Reply by kabechiyo** [reply]
@Yarakashi-Kikohshi 
お待たせしました！
記事内の脚注が見やすくなるように、テキストの前後にブラケット（[]）を追加しました。
https://qiita.com/release-notes/2024-06-06

**Reply by yarakos95** [reply]
@kabechiyo 
対応ありがとうございます 🎊
脚注番号の前後に角括弧が付くようになったことを確認しました！

---

重箱の隅をつつくようで悪いのですが、今回の角括弧の付き方は脚注番号以外にも及んでいます。そのため、一部のユーザーにとって意図しない変更になっている可能性があります。

これは、リンクを `<sup>` タグで囲った場合に発生します。以下の例では、リンクとなっている `[example](https://www.example.com)` に角括弧は付かないはずです。

```markdown
これは脚注 [^1][^2]

[^1]: 1 つ目の脚注
[^2]: 2 つ目の脚注

これは `<sup>` されたリンク <sup>[example](https://www.example.com)</sup>
```

![image](https://github.com/increments/qiita-discussions/assets/75878492/f8d0384f-adb1-4a53-a65f-1ffc7f5ffec2)

例えば、GitHub 上では、同じようにしても `[example](https://www.example.com)` に角括弧は付きません。

これは脚注 [^1][^2]

[^1]: 1 つ目の脚注
[^2]: 2 つ目の脚注

これは `<sup>` されたリンク <sup>[example](https://www.example.com)</sup>

\# これは要求ではなくただの気付きですが、リンクには下線があって脚注番号には下線が無いのですね。

**Reply by kabechiyo** [reply]
@Yarakashi-Kikohshi 

ご確認ありがとうございます！🙌

> 今回の角括弧の付き方は脚注番号以外にも及んでいます。そのため、一部のユーザーにとって意図しない変更になっている可能性があります。
> これは、リンクを `<sup>` タグで囲った場合に発生します。以下の例では、リンクとなっている `[example](https://www.example.com/)` に角括弧は付かないはずです。

ご報告いただきありがとうございます！
こちら意図しない挙動でしたので、修正いたします。

> これは要求ではなくただの気付きですが、リンクには下線があって脚注番号には下線が無いのですね。

脚注は下線がない方が見やすいと判断し、このようにしておりました！

**Reply by kabechiyo** [reply]
@Yarakashi-Kikohshi 

お待たせしました！
こちらリンクを `<sup>`タグで囲った際には角括弧がつかないように修正いたしました。

**Reply by yarakos95** [reply]
@kabechiyo 

`<sup>` されたリンクに角括弧が付かないことを確認しました。
対応ありがとうございます 🎉
